### ISSUE TITLE
Store is not closed!

### ISSUE THREAD

**Issue by erdalp** [smartstore/Smartstore#664]
I moved my project running on Smartstore v5.0.3 to a different server. When I try to close the store on the new server, the store does not close. In the same way, I did not understand why the tax-inclusive prices of the products work on the old server, but not on the new server. Is there a bug or do I need to configure something?

https://user-images.githubusercontent.com/20066079/233945091-2b462c6c-febd-4f34-ae8d-b62249151a3b.mp4

**Comment by erdalp** [comment]
Unfortunately, there is no log entry generated for this. There is no record in either the project's logs or the Windows logs.

**Comment by muratcakir** [comment]
Like Michael already said, the problem is not reproducible. Your video implies that you have done some customizations (in the backend). You will have to debug and find out what's going on.

**Comment by erdalp** [comment]
Thank you very much for your support. I solved the problem.

I followed a way like this. First of all, I thought that the problem might be due to antivirus restrictions on the server, so I uninstalled the antivirus.

Later, when I debugged the project, I noticed an error like this; Could not retrieve data with ICacheManager. When I examined this in more detail, it made me think that it was due to the fact that I moved the database to the new server in the form of Scripts and did not receive the indexes.

Therefore, when I did the operation again, the error was resolved.
